execution_engine_testing/tests/Cargo.toml

```diff
@@ -36,7 +36,9 @@ walrus = "0.19.0"
 wat = "1.0.47"
 
 [features]
+default = []
 use-as-wasm = ["casper-engine-test-support/use-as-wasm"]
+fixture-generators = []
```

Comment:
I'd rather avoid adding undocumented new features. Do we really need these changes?

Reply:
I think it's useful to keep the code that generates lmdb fixtures in the code, but I don't see a better way other than commenting the code, or removing the fixture generator

Reply:
Good point, removed the feature flag and added `RUN_FIXTURE_GENERATORS` env var (although it's unfortunate you can't skip/ignore tests based on env var)